packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -1014,6 +1026,81 @@ export const useArbTokenBridge = (
     )
   }
 
+  // mutates provided array - assigns deadlineBlockNumbers to all asserted but unconfimred l2tol1 messages 
+  const addNodeDeadlineToUnconfirmedWithdrawals = async(l2ToL1Data: L2ToL1EventResultPlus[])=>{
+    if(l2ToL1Data.length === 0) return []
+    const l1NetworkID = await l1NetworkIDCached()
+    const oldOutboxOffset = (()=>{
```

Comment:
Could add a quick comment saying the offset is needed because of the outbox v1 to v2 transition

Reply:
Could also do a ternary operator instead of IIFE here, could be shorter cleaner

Reply:
```suggestion
    if(l1NetworkID !== '1' && l1NetworkID !== '4') throw err
    const oldOutboxOffset = l1NetworkID === 1 ? 30 : 326
```

Reply:
fiiiine